src/ui.js

```diff
@@ -6551,8 +6582,14 @@ document.addEventListener("DOMContentLoaded", () => {
     ui.initialise();
 
     const load_quiver_from_query_string = () => {
-        // If there is `q` parameter in the query string, try to decode it as a diagram.
         const query_data = query_parameters();
+
+        // If there is `e` in the query string, we want to treat the UI as running in embedded mode.
+        if (query_data.has("e")) {
```

Comment:
I think it could also be worth checking for this in `index.html` and adding a class to the body so that we can override the style of `.loading-screen` and change the background in that case to `background: white;`, as the majority of web pages will be displaying these diagrams on white backgrounds, in which case the dark loading screen looks incongruous.